clients/spatial_relations.py

```diff
@@ -109,6 +120,24 @@ def isclose(bb1, bb2):
 
     return dist < 2 * dim2
 
+def isin(bb1, bb2):
+
```

Comment:
Could you add a bit of doc? at least to mention that:
- the function checks whether bb1 in is bb2 (and not the contrary)
- the computation is based on checking whether bounding box A is contained in bounding box B